### ISSUE TITLE
complex input

### ISSUE THREAD

**Issue by v4nz666** [v4nz666/RoguePy#27]
Allow for input triggers like "SHIFT+TAB".

Currently, chars bound to input will accept either upper or lower case, should this be the case?

**Comment by capnmikey** [comment]
I don't think this should be the case, no. In most rogue-likes you have a gazillion commands, with most common ones being letters, and less common ones being shift-letters. It'd be easy enough (albeit a little tedious) to bind both upper- and lower-case characters to the same command if you wanted that behaviour.
